css/index.css

```diff
@@ -112,6 +175,110 @@ p {
     justify-content: center;
 }
 
```

Comment:
The .middle-content .boxes .box class has all the styles required to style the individual boxes. Instead of typing out all the redundant CSS again, you can change your HTML in the index.html.  The original HTML was `<div class="box">` You can now add another class to it or an id like so `<div class="box .box1">`  This will take advantage of parent/child relationships and the child will inherit the parent styles and you can just override the color for each box.